.gitignore

```diff
@@ -2,16 +2,10 @@
 
 # project local files
 *.local.ts
-
-# compiled output
-/dist
-/tmp
-/out-tsc
-# Only exists if Bazel was run
-/bazel-out
+*.local.txt
```

Comment:
May be use `*.local.*`?

Reply:
you can't have a `*` for extensions on gitignore